addons/skin.estuary/xml/Variables.xml

```diff
@@ -661,4 +661,7 @@
 		<value condition="VideoPlayer.SubtitlesEnabled">[B]$INFO[VideoPlayer.SubtitlesLanguage][/B]</value>
 		<value>[B]$LOCALIZE[1223][/B]</value>
 	</variable>
+	<variable name="PVRInfoParentalRatingCode">
+		<value condition="!String.IsEmpty(ListItem.parentalratingcode)"> [B][COLOR green]($INFO[ListItem.parentalratingcode])[/COLOR][/B]</value>
```

Comment:
Adding colour green to a rating might be misleading. I'd avoid colouring all together and maybe show it decoupled from the genre or it might be misleading as well

Reply:
@enen92 - Thanks for your input.

The first reason that I coupled the parental rating with the genre is that they are logically related.  A parent may say no to a 'Horror' movie, but yes to the same 'Horror (G)' movie.  Likewise, they may say no to a 'Comedy (R)' movie but maybe accidentally yes to the same 'Comedy' movie.

This would proximity also be beneficial if one day Kodi was able to show parental rating icons from the source system instead of just text.  (Something that I am looking into.)

The second reason was to preserve screen real estate.  Adding a new line and label for the parental rating seemed like a waste of space, especially in the 'MyPVRGuide' view.  There are already several other fields that are strung together onto a single line within this skin.

As for the colour: the (bracket) and colour were to add a small amount of visual separation between the genre and the parental rating.  Green was just a design choice to make it stand out, it could have been anything.

Would you agree to a design compromise?  I make the parental rating code the same dark grey as the labels (not the light grey of the values), but leave it in-line.  Being the same colour as a label would indicate that it is a different field and would suggest that the value displayed is its own label.

Reply:
I kinda like the color used as a visual separator, I do the same concept here: https://kodi.wiki/view/Game_add-ons

The colors are of course arbitrary, but the idea is that young kids is green, age 13+ is yellow, and adult-only is red. Color is a way to include a visual separate, in a way that doesn't really matter what the color is, yet is still an indicator to those paying attention.

Reply:
@garbear - Thanks for your feedback.

I have become aware that some regions use a traffic light system, green/amber/red, to indicate age suitability, so a green '(R)' may be misleading.  It was not my intent to replicate or imply this and I agree that a more neutral colour is desirable.

As part of future development in this area, if icons are added (supplied by the back end), then they can be coloured according to the local laws of that region.

Australia, Brazil, Cyprus, Denmark, Germany, the list goes on, all use coloured icons as part of their classification scheme.

Reply:
> Would you agree to a design compromise? I make the parental rating code the same dark grey as the labels (not the light grey of the values), but leave it in-line. Being the same colour as a label would indicate that it is a different field and would suggest that the value displayed is its own label.

@enen92 - Do you have any thoughts on this proposal?

Reply:
I am fine if the color is removed, not a big deal personally to show it next to the genre. 
However, some review from our PVR guys would be nice (@ksooo @phunkyfish)

Reply:
In the screenshots the title colour would work to differentiate it without picking colours.

Reply:
The requested changes have been applied.

![screenshot00004](https://github.com/xbmc/xbmc/assets/127641886/33f881ee-b609-4118-93b6-501796c1f095)

![screenshot00005](https://github.com/xbmc/xbmc/assets/127641886/b11642c6-3b2d-4b3a-a7dc-c668af2b289c)